app/views/insured/families/_enrollment_refactored.html.erb

```diff
@@ -1,5 +1,12 @@
 <% product = hbx_enrollment.product %>
 
+<% if @year_display != hbx_enrollment.effective_on.year && EnrollRegistry.feature_enabled?(:home_tiles_group_by_year) %>
```

Comment:
This logic feels like it should be a level up, in the parent file, as it's more about how the tiles are shown collectively on the page as opposed to within each tile itself. Is there a reason this logic is here instead of in home.html.erb?

I'm also wondering if this will collide with the logic on the enrollments page that does similar categorizing and potentially duplicate year labels. Could also break bc @year_display is not set on this history page.